sdk/search/Azure.Search.Documents/src/Batching/Publisher.cs

```diff
@@ -289,6 +289,8 @@ protected virtual async Task OnDocumentsAddedAsync(IEnumerable<T> documents, Can
                 _pending.Enqueue(action);
             }
 
+            AzureSearchDocumentsEventSource.Instance.PendingQueueResized(_pending.Count);
```

Comment:
I worry about the wording of these.  We have an implicit notion of a batch size that we scale.  By saying the pending queue has been resized vs. saying pending index actions have been added, I think it confuses that.